Gemfile

```diff
@@ -22,4 +22,4 @@ gem 'rubocop-rake', '~> 0.6'
 gem 'simplecov', require: false
 gem 'with_advisory_lock'
 
-gem 'mermaid', '>= 0.0.5'
+gem 'mermaid', '>= 0.0.6'
```

Comment:
It requires to have mermaid also in the :
https://github.com/seuros/rails_lens/blob/master/rails_lens.gemspec

Otherwise i am getting:
```
cannot load such file -- mermaid (LoadError)
	from /Users/navid/.local/share/mise/installs/ruby/3.4.5/lib/ruby/gems/3.4.0/gems/rails_lens-0.2.7/lib/rails_lens/erd/visualizer.rb:3:in '<main>'
```